dashboard/src/components/resource-list/ResourceListView.js

```diff
@@ -467,7 +467,7 @@ const NoDataMessage = ({ type }) => {
   const classes = useStyles();
 
   function redirect() {
-    window.location.href = "https://docs.featureform.com/quickstart";
+    window.location.href = "https://docs.featureform.com/getting-started/overview";
```

Comment:
In the future, should just put this is its own pr

Reply:
I can remove the change and open a new PR with just this change.

Reply:
Don't bother